### PR TITLE
Finish video feature v2

### PR DESCRIPTION
**Pull request by oscarlopez8**

### CODE REVIEW

File: src/main/java/com/jalasoft/jfc/controller/VideoConverterController.java [hunk 1]
```diff
@@ -76,7 +76,7 @@ public Response videoConverter(
             @RequestParam(defaultValue = "0") int height, @RequestParam(defaultValue = "") String videoCodec,
             @RequestParam(defaultValue = "") String videoBitRate, @RequestParam(defaultValue = "-1") int quality,
             @RequestParam(defaultValue = "") boolean isThumbnail, @RequestParam(defaultValue = "false")
-            boolean isMetadata, HttpServletRequest request) {
+            boolean isMetadata, HttpServletRequest request, @RequestParam(defaultValue = ".png") String videoFormat) {
```
Comment (Amet015) [inline src/main/java/com/jalasoft/jfc/controller/VideoConverterController.java]: a format video .png?

File: src/main/java/com/jalasoft/jfc/model/video/VideoConverter.java [hunk 2]
```diff
@@ -14,8 +14,10 @@
 import com.jalasoft.jfc.model.Param;
 import com.jalasoft.jfc.model.command.ffmpeg.CommandVideoBitRate;
 import com.jalasoft.jfc.model.command.ffmpeg.CommandVideoCodec;
+import com.jalasoft.jfc.model.command.ffmpeg.CommandVideoFormat;
 import com.jalasoft.jfc.model.command.ffmpeg.CommandVideoFrameRate;
 import com.jalasoft.jfc.model.command.ffmpeg.CommandVideoScale;
+import com.jalasoft.jfc.model.command.imagick.CommandImageFormat;
```
Comment (JuanCarlosMartinez4) [inline src/main/java/com/jalasoft/jfc/model/video/VideoConverter.java]: delete this import

File: src/main/java/com/jalasoft/jfc/model/video/VideoParam.java [hunk 3]
```diff
@@ -193,4 +204,12 @@ public void setThumbnail(boolean thumbnail) {
     public void setVideoFrame(byte videoFrame) {
         this.videoFrame = videoFrame;
     }
+
+    /**
+     * Sets videoFrame value.
+     * @param videoFormat value.
```
Comment (JuanCarlosMartinez4) [inline src/main/java/com/jalasoft/jfc/model/video/VideoParam.java:210]: update this comment

File: src/main/java/com/jalasoft/jfc/model/video/VideoParam.java [hunk 4]
```diff
@@ -193,4 +204,12 @@ public void setThumbnail(boolean thumbnail) {
     public void setVideoFrame(byte videoFrame) {
         this.videoFrame = videoFrame;
     }
+
+    /**
+     * Sets videoFrame value.
```
Comment (JuanCarlosMartinez4) [inline src/main/java/com/jalasoft/jfc/model/video/VideoParam.java]: update this comment

**Review by JuanCarlosMartinez4 — APPROVED** [review]

**Pull request merged**